### PR TITLE
artifactory 4.3.2

### PR DESCRIPTION
**Pull request by jainishshah17**

### CODE REVIEW

File: Library/Formula/artifactory.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 class Artifactory < Formula
   desc "Manages binaries"
   homepage "https://www.jfrog.com/artifactory/"
-  url "https://dl.bintray.com/jfrog/artifactory/jfrog-artifactory-oss-4.0.2.zip"
-  sha256 "52a882b3681bc00eb73fddfe6abdc739156c2940be073406185be23a5f35bff2"
-
-  bottle :unneeded
```
Comment (mikemcquaid) [inline Library/Formula/artifactory.rb]: Please leave this line as-is, as previously discussed.

Comment (jainishshah17) [inline Library/Formula/artifactory.rb]: What do you want me to Change here?

Comment (mikemcquaid) [inline Library/Formula/artifactory.rb]: Re-add `bottle :unneeded`

Comment (jainishshah17) [inline Library/Formula/artifactory.rb]: When we will be able to see this change?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/f4eb8eb377ae7815254916168b23bd41d91d5ef3. Thank you for your contribution to Homebrew @jainishshah17; we appreciate it! :smiley_cat:

**Pull request closed without merging**